### ISSUE TITLE
[OT] - Thank you

### ISSUE THREAD

**Issue by elcuervo** [soveran/cuba#57]
Hi! I'm one of the many users that love Cuba. I also have the pleasure to know you and enjoy all of your work and products but now I want to show my gratitude in a public manner. 

This OT and useless issue it's just my dumb way to say thank you for the work you do, the support to the community and the lesscode initiative that makes a bunch of us happy.

Next time I'll buy you a coffee and give you a hug.

**Comment by clintpachl** [comment]
Bruno Aguirre wrote, On 01/09/15 14:01:
>
> Hi! I'm one of the many users that love Cuba. I also have the pleasure 
> to know you and enjoy all of your work and products but now I want to 
> show my gratitude in a public manner.
>
> This OT and useless issue it's just my dumb way to say thank you for 
> the work you do, the support to the community and the lesscode 
> initiative that makes a bunch of us happy.
>
> Next time I'll buy you a coffee and give you a hug.
>

I second Bruno.

I have switched Ruby frameworks over the years since 2005 starting with 
Rails then Ramaze then Sinatra. Over the last year or two I reviewed all 
the Ruby frameworks available. When I first saw Cuba, I didn't 
understand it for some reason and never gave it a chance. So I continued 
on with Sinatra.

Then recently I listened to Michel Martens (soveran) on Ruby Rogues 
podcast. He intrigued me with his minimalist mindset. Prior to web 
programming, I programmed hard-real-time embedded systems, so I 
appreciate minimalism and speed. So I decided to take a second look at 
Cuba. After checking out the slides on Cuba.is and the code on Github, I 
was convinced Cuba was the perfect framework for me.

I just wished I would have "got it" the first time around. I guess not 
having routes directly associated with an HTTP verb (didn't seem 
RESTful), not to mention route nesting, just didn't make sense to me at 
the time. Now I see its true genius.

Anyway, as time permits, I will happily convert my Sinatra apps to Cuba. 
I look forward to joining the Cuba community.

Thanks to Christian Neukirchen, Mr. Martens, and all contributors to the 
Rack and Cuba projects.

Clint
